FED.md

```diff
@@ -0,0 +1,67 @@
+FED Ways of Working
+GOAL: Maintenance, Performance, and Readability.
+
+HTML
+-  We use SMACSS, BEM methodologies to make HTML modular
+
+SMACSS Structure
+https://smacss.com/book/
+
+PATH: iogt/clients/css/
```

Comment:
Should this say something other than `iogt`?

Nitpick: should we use `client` instead of `clients`? Not too big a deal though.